OPHD/Population/PopulationTable.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <array>
+
+struct PopulationTable
+{
+public:
+	enum class Role
+	{
+		Child,
+		Student,
+		Worker,
+		Scientist,
+		Retired
+	};
+
+	int& operator[](std::size_t);
+	int operator[](std::size_t) const;
+
+	int& operator[](Role);
+	int operator[](Role) const;
```

Comment:
@DanRStevens, if you have some time, I would appreciate checking these 4 container access operator overloads to make sure I applied them well. The code is working when testing. Having the extra overloads allows accessing via enum class Role or std::size_t without applying static_casts in code.